features/step_definitions/planning_application_steps.rb

```diff
@@ -98,17 +98,6 @@
   )
 end
 
-Given("the planning application is determined") do
```

Comment:
Removed cucumber with 'Review assessment' without breaking tests
  - the link 'Review assessment' is being changed so I was testing
    out the code but this didn't break anything so I've removed it